client/secureclient/src/test/java/com/linkedin/openhouse/client/ssl/TablesApiClientFactoryTest.java

```diff
@@ -8,13 +8,34 @@
 import com.linkedin.openhouse.tables.client.invoker.ApiClient;
 import io.netty.handler.ssl.SslContext;
 import java.io.File;
+import java.io.IOException;
 import java.net.MalformedURLException;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
 import org.junit.jupiter.api.Test;
 import org.mockito.ArgumentCaptor;
 import org.mockito.Mockito;
 
 public class TablesApiClientFactoryTest {
 
+  static File tmpCert;
+
+  static TablesApiClientFactory tablesApiClientFactorySpy;
+
+  @BeforeAll
```

Comment:
taking these methods from the single test below and bringing them up to shared methods because now two unittests use these mocks/spys.